hedging/tests/fixtures/hedging.json

```diff
@@ -10,4 +10,4 @@
       "liability": "0"
     }
   ]
-}
+}
```

Comment:
You need to fix your editor to add new lines at end of file.

Reply:
I see, I was wondering about this... thx